src/components/KSelect/KSelect.vue

```diff
@@ -238,435 +255,407 @@ export interface SelectFilterFnParams {
 
 export type DropdownFooterTextPosition = 'sticky' | 'static'
 
-export default defineComponent({
-  name: 'KSelect',
-  components: {
-    KButton,
-    KIcon,
-    KInput,
-    KLabel,
-    KPop,
-    KSelectItems,
-    KSelectItem,
-    KToggle,
+</script>
+
+<script setup lang="ts">
+const { getSizeFromString } = useUtilities()
+
+const defaultKPopAttributes = {
+  popoverClasses: 'k-select-popover mt-0',
+  popoverTimeout: 0,
+  placement: 'bottomStart',
+  hideCaret: true,
+}
+
+const props = defineProps({
+  modelValue: {
+    type: [String, Number],
+    default: '',
   },
-  inheritAttrs: false,
```

Comment:
@adamdehaven here

Reply:
Ah, I missed it in the refactor 👍🏼

Reply:
Yeah, the PR is packed with changes - sorry for that, blame it on `script setup`